recipes/ios/src/ios_browser.m

```diff
@@ -11,3 +11,30 @@ void ios_open_url(char *url)
 	NSString *nsurl = [NSString stringWithCString:(char *)url encoding:NSUTF8StringEncoding];
 	[[UIApplication sharedApplication] openURL:[NSURL URLWithString: nsurl]];
 }
+
+/*
+ * Webview support
```

Comment:
Need some more docs on how to use the implementation, a small example would help here.